bondcpp/package.xml

```diff
@@ -1,4 +1,6 @@
-<package>
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
```

Comment:
Thanks @wkentaro for reporting. This looks like a problem to report upstream given that this package uses the format defined in [REP140](http://www.ros.org/reps/rep-0140.html#schema).
So it looks more like a rospack issue: this looks to be related https://github.com/ros/rospack/issues/77.

Given that it works on xenial but not the later platforms my guess is that it's a tinyxml2 issue. They seem to have bumped the version significantly between xenial and zesty
```
libtinyxml2-dev | 2.2.0-1.1ubuntu1          | xenial/universe           | amd64, arm64, armhf, i386, powerpc, ppc64el, s390x
libtinyxml2-dev | 4.0.1-1                   | zesty/universe            | amd64, arm64, armhf, i386, ppc64el, s390x
```